README.md

```diff
@@ -1 +1,60 @@
-# my-portfolio
\ No newline at end of file
+# my-portfolio
+![](https://img.shields.io/badge/myapp-blueviolet)
+
+
+> Personal portfolio project.
+
+<!-- ![screenshot](./app_screenshot.png) -->
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## Live Demo
+
+[Live Demo Link](https://github.com/dami1080/my-portfolio/tree/features)
+
+## Instructions
+
+To get a local copy up and running follow these simple example steps.
+
+$ cd <folder>
+
+$ https://github.com/dami1080/my-portfolio.git 
```

Comment:
- [ ] **OPTIONAL** Kindly keep in mind that there might be non-technicals who are going to view your project in the future. So, the README needs to be human-readable.
Each command line (CLI) needs to be explained to the reader. Also, the second CLI is `git clone <repo's link>`.
To use the markup language for CLIs you can wrap it inside these `` to be like the one above ⬆️